### ISSUE TITLE
No allowance for focus

### ISSUE THREAD

**Issue by inghamn** [City-of-Bloomington/factory-number-one#6]
Currently, you cannot use the keyboard to navigate the screen, since we do not highlight what elements are currently selected.

To make application keyboard accessible, we should have CSS for focus.  That way folks can tab through screens and see which item is selected.
